setup.py

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+import re
+from pathlib import Path
+from typing import List
+
+import setuptools
+
+
+def read_requirements(path: str) -> List[str]:
+    with open(path) as f:
+        return [line for line in f.readlines() if not line.startswith("#")]
+
+
+version = re.search(r'__version__\s*=\s*"(.*?)"', Path("utils/__init__.py").read_text())
+assert version is not None, "could not parse version!"
+
+with open("README.md", "r", encoding="utf-8") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="utils",
```

Comment:
That's gonna conflict with https://pypi.org/project/utils/

Reply:
Wasn't planning to, at least initially. We can install directly from git.

Reply:
pip is gonna complain if someone installs that package.

Reply:
Our components run in their own container / own Python environment. So I doubt it'll affect us. We can name it `granulate-utils`.

Reply:
Let's make this change if this ever bothers

Reply:
Changed in #6